src/nools/lib.js

```diff
@@ -2,9 +2,7 @@ for(idx1=0; idx1<targets.length; ++idx1) {
   target = targets[idx1];
   switch(target.appliesTo) {
     case 'contacts':
-      if(c.contact && target.appliesToType.indexOf(c.contact.type) !== -1) {
-        emitContactBasedTargetFor(c, target);
-      }
+      emitContactBasedTargetFor(c, target);
```

Comment:
I thought `emitContactBasedTargetFor` and `emitReportBasedTargetFor` were being merged into `emitTargetFor` by
https://github.com/medic/medic-conf/pull/156
Wouldn't it make sense to put these changes in there as well?

Reply:
Done